### ISSUE TITLE
Navigation and main area show collections and their record numbers but clicking on them does nothing

### ISSUE THREAD

**Issue by LukasDeco** [gterrono/houston#393]
This is the error I get: 
TypeError: Meteor.autorun is not a function
    at Object.Meteor.subscribeWithPagination (tmeasday_paginated-subscription.js?hash=1dd8722609ac26b4a8d4e83b40fec2f492b1ea8f:93)
    at setup_collection (router.coffee:21)
    at Object.subs (router.coffee:64)
    at ctor.options.waitOn (router.coffee:37)
    at iron_router.js?hash=f37bffce0ac91b7fda73e7fed6ffa6ad745b41fc:435
    at Array.forEach (<anonymous>)

It shows collections and document numbers, but clicking on them does nothing and yields this error. I read somewhere that Meteor.autorun is deprecated?

**Comment by lucnat** [comment]
Yes, it uses `Meteor.autorun` which has been removed, it is now called `Tracker.autorun`. You can fix this by doing on the client on startup:
`Meteor.autorun = Tracker.autorun;`
And the admin pannel will work again :)

**Comment by listcrawler-houston** [comment]
> Yes, it uses `Meteor.autorun` which has been removed, it is now called `Tracker.autorun`. You can fix this by doing on the client on startup: `Meteor.autorun = Tracker.autorun;` And the admin pannel will work again :)

Yay this worked, thanks
